src/harness/certs/generate_fake_certs.sh

```diff
@@ -151,6 +167,77 @@ openssl ca -cert unknown_ca.cert -keyfile private/unknown_ca.key -in unknown_dev
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
 
+# Generate certificates for test case WINNF.FT.S.SCS.6 - Unrecognized root of trust certificate presented during registration
+echo "\n\nGenerate 'unrecognized_device' certificate/key"
+openssl req -new -x509 -newkey rsa:4096 -sha384 -nodes -days 7300 \
+    -extensions root_ca -config ../../../cert/openssl.cnf \
+    -out unrecognized_root_ca.cert -keyout private/unrecognized_root_ca.key \
+    -subj "/C=US/ST=CA/L=Somewhere/O=Generic Certification Organization/OU=www.example.org/CN=Generic RSA Root CA"
+
+openssl req -new -newkey rsa:2048 -nodes \
+    -reqexts cbsd_req -config ../../../cert/openssl.cnf \
+    -out unrecognized_device.csr -keyout unrecognized_device.key \
+    -subj "/C=US/ST=CA/L=Somewhere/O=Generic Certification Organization/OU=www.example.org/CN=Unrecognized CBSD"
+openssl ca -cert unrecognized_root_ca.cert -keyfile private/unrecognized_root_ca.key -in unrecognized_device.csr \
+    -out unrecognized_device.cert -outdir ./root \
+    -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
+    -batch -notext -create_serial -utf8 -days 1185 -md sha384
+
+# Certificates for test case WINN.FT.S.SCS.7 - corrupted certificate, based on dp_client.cert
+key="client.key"
+cert="client.cert"
+outcert1="corrupted_client.key"
+outcert2="corrupted_client.cert"
+gen_corrupt_cert $key $cert $outcert1 $outcert2
+
+#Certificate for test case WINNF.FT.S.SCS.8 - Self-signed certificate presented during registration
+#Using the same CSR that was created for normal operation
+echo "\n\nGenerate 'self_signed_client' certificate/key"
+openssl x509 -signkey client.key -in client.csr \
+    -out self_signed_client.cert \
+    -req -days 1185
+
+#Certificates for test case WINNF.FT.S.SCS.9 - Non-CBRS trust root signed certificate presented during registration
+echo "\n\nGenerate 'non_cbrs_root_ca' certificate/key"
+openssl req -new -x509 -newkey rsa:4096 -sha384 -nodes -days 7300 \
+    -extensions root_ca -config ../../../cert/openssl.cnf \
+    -out non_cbrs_root_ca.cert -keyout private/non_cbrs_root_ca.key \
+    -subj "/C=US/ST=District of Columbia/L=Washington/O=Wireless Innovation Forum/OU=www.wirelessinnovation.org/CN=WInnForum RSA Root CA-2"
+
+#Generate CBSD CA certificate signed by non_cbrs_root_ca
```

Comment:
perhaps remove this comment /echo, to keep only a header comment/echo for each testcase